test/goose/test_utils.clj

```diff
@@ -55,6 +58,71 @@
 
   (clear-redis))
 
+(def called? (atom false))
+(defn get-called? [] @called?)
+(defn set-called? [value] (reset! called? value))
+
+(defn reset-called?-fixture
+  [f]
+  (set-called? false)
+  (f))
+
+(defn stub [f]
+  (fn [& args]
+    (set-called? true)
+    (apply f args)))
```

Comment:
Managing this atom will get out of hand quickly. For example, the tests can only have 1 stub each. You'd soon want richer stub behavior. I'd suggest using a full-fledged library if you are going the stubbing route https://github.com/alexanderjamesking/spy